src/server/common/model/answer/on-off-farm.js

```diff
@@ -45,6 +47,16 @@ export class OnOffFarm extends AnswerModel {
     }
   }
 
+  /** @param {AnswerErrors} errors */
+  static errorMessages(errors) {
```

Comment:
Are we not supposed to use the static method defined already in the answer model?

Reply:
no as validation might be slightly different on each answer (one field vs many) so base throws a "not Implemented" error